includes/Checker/Checks/Plugin_Repo/Plugin_Readme_Check.php

```diff
@@ -147,7 +147,7 @@ private function check_name( Check_Result $result, string $readme_file, Parser $
 				9
 			);
 		} else {
-			$plugin_data = get_plugin_data( $result->plugin()->main_file() );
+			$plugin_data = get_plugin_data( $result->plugin()->main_file(), true, false );
```

Comment:
Do we really need the markup?

```suggestion
			$plugin_data = get_plugin_data( $result->plugin()->main_file(), false, false );
```